awx/main/tests/unit/api/test_views.py

```diff
@@ -110,13 +110,13 @@ def exclude(self, **kwargs):
 
         with mocker.patch.object(InventoryInventorySourcesUpdate, 'get_object', return_value=obj):
             with mocker.patch.object(InventoryInventorySourcesUpdate, 'get_serializer_context', return_value=None):
-                with mocker.patch('awx.api.serializers.InventoryUpdateDetailSerializer') as serializer_class:
-                    serializer = serializer_class.return_value
-                    serializer.to_representation.return_value = {}
+                serializer_class = mocker.patch('awx.api.serializers.InventoryUpdateDetailSerializer')
+                serializer = serializer_class.return_value
+                serializer.to_representation.return_value = {}
 
-                    view = InventoryInventorySourcesUpdate()
-                    response = view.post(mock_request)
-                    assert response.data == expected
+                view = InventoryInventorySourcesUpdate()
+                response = view.post(mock_request)
+                assert response.data == expected
```

Comment:
@jessicamack why was this changed

Reply:
The tests were throwing an error and this warning about using mocks as a context manager https://pytest-mock.readthedocs.io/en/latest/remarks.html#usage-as-context-manager so I reformatted the tests.

Reply:
A long shot, but could this have been related to the test that we eventually disabled? Can you try reverting these changes and then run tests a) with that naughty test disabled and b) with the naughty test enabled? If we can get past tests with this change reverted, and maybe even get past the other test with this change reverted, then we should revert it.